src/mjolnir/statistics.h

```diff
@@ -1,20 +1,25 @@
 #ifndef VALHALLA_MJOLNIR_STATISTICS_H_
 #define VALHALLA_MJOLNIR_STATISTICS_H_
 
+#include "baldr/graphconstants.h"
+#include "midgard/aabb2.h"
+
+#include <boost/property_tree/ptree.hpp>
+
 #include <map>
 #include <string>
 #include <unordered_map>
 #include <unordered_set>
 #include <vector>
 
-#include "baldr/graphconstants.h"
-#include "midgard/aabb2.h"
-#include <boost/property_tree/ptree.hpp>
-
 struct sqlite3_stmt;
 
-using namespace valhalla::baldr;
+namespace valhalla {
+namespace mjolnir {
+
 using namespace valhalla::midgard;
+using ::valhalla::baldr::RoadClass;
```

Comment:
No idea why, but somehow clang20 produced an error without that change saying something like "cannot use `valhalla::baldr::RoadClass` where `valhalla::RoadClass` is expected"...

Reply:
yeah strange.. `valhalla::RoadClass` is coming from the proto definitions, but shouldn't play a role in this header it seems. it's unfortunate that lots of proto definitions have the exact same name as the C++ equivalents.

Reply:
another good reason to never use google code style and stick to the stl style of `snake_case`